js/ui/themes.js

```diff
@@ -178,12 +179,14 @@ function init(options) {
     processMarkup();
 
     currentThemeName = undefined;
+    materialTheme = undefined;
     current(options);
 }
 
 function current(options) {
     if(!arguments.length) {
         currentThemeName = currentThemeName || readThemeMarker();
+        setMaterialThemeSign();
```

Comment:
themes.current() is a getter. why do we need to set 'materialTheme' variable in getter?